comps/llms/text-generation/vllm-openvino/build_vllm_openvino.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright (c) 2024 Intel Corporation
```

Comment:
This license header is out of date, please leave it empty (the bot will add it automatically) or add the new one,   
```
Copyright (C) 2024 Intel Corporation   
SPDX-License-Identifier: Apache-2.0   
```